railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -77,6 +79,7 @@ def db_migrate_and_schema_cache_dump
       def db_migrate_and_schema_cache_dump_and_schema_cache_clear
         Dir.chdir(app_path) do
           generate_models_for_animals
+          rails "db:create"
```

Comment:
I'm guessing all of the tests in this file need to prepend "db:create" task, see also:
https://github.com/rails/rails/blob/6a28970bba2ea919ce161275f945d5ee08ad8e18/railties/test/application/rake/multi_dbs_test.rb#L204